test.spec.js

```diff
@@ -25,10 +30,12 @@ module.exports = {
   testHash (assert) {
     'use strict';
 
-    assert.expect(1);
+    assert.expect(3);
 
     argon2.hash(password, salt).then((hash) => {
-      assert.equal(hash, '$argon2i$m=4096,t=3,p=1$c29tZXNhbHQ$vpOd0mbc3AzXEHMgcTb1CrZt5XuoRQuz1kQtGBv7ejk');
+      assert.ok(hash, 'Hash should be defined.');
+      assert.ok(/\$argon2i\$/.test(hash), 'Should have argon2i signature.');
+      assert.equal(hash, hashes.argon2i);
```

Comment:
The test was like that before but that's very much redundant. If the hash is equal to the expected sample, then it certainly will have the `$argon2i$` signature and absolutely will be defined, as `undefined` returns `false` for every regex test.